app/assets/stylesheets/components/_postcode-lookup.scss

```diff
@@ -0,0 +1,5 @@
+.postcode-lookup {
```

Comment:
This file is in `/components/` but this code change isn't a component - it should be somewhere else.

Reply:
Could it go in `app/assets/stylesheets/views`?

Reply:
I've moved it to views but it's probably not the best place long term. I will bring this up again in the frontend community regarding patterns